docs/src/main/sphinx/connector/opensearch.md

```diff
@@ -112,6 +112,8 @@ following options must be configured:
     the configured IAM user must be able to assume this role.
 * - `opensearch.aws.external-id`
   - Optional external ID to pass while assuming an AWS IAM role.
+* - `opensearch.aws.deployment-type`
+  - AWS OpenSearch deployment type. This option is required.
```

Comment:
It would be nice to mention the possible values (PROVISIONED & SERVERLESS). Otherwise, readers have to look for the config class.

Reply:
sure, added the possible values.